extensions/firefox/content/PdfJs.jsm

```diff
@@ -173,7 +173,7 @@ var PdfJs = {
 
   updateRegistration: function updateRegistration() {
     if (this.enabled) {
-      this._ensureRegistered();
+      this.ensureRegistered();
```

Comment:
If this one is public, i.e., has no underscore, then I think it would be best to make `_ensureUnregistered` public as well, so remove the underscore there as well for consistency.